Enhanced Barter.user.js

```diff
@@ -163,6 +163,9 @@ function barterReady() {
                     <small id="ggdeals_${barterId}_after"></small>
                 </span>`);
         $(`[id="ggdeals_${spanId}"]`).click(async () => {
+                // inform user as this can take a bit with calls to barter
+                $(`[id="ggdeals_${barterId}_after"]`).html(` (<a title="GG.Deals current lowest price (Official Stores / Keyshops)">Loading Price please wait</a>)`);
```

Comment:
```suggestion
                $(`[id="ggdeals_${barterId}_after"] a`).text(`Loading price, please wait`);
```

Reply:
I will omit the a as the id should be sufficient and adding the a makes the selector not hit.